srcpkgs/mednaffe/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'mednaffe'
+pkgname=mednaffe
+version=0.8.8
+revision=1
+build_style=gnu-configure
+configure_args="--enable-gtk3"
+hostmakedepends="autoconf pkg-config"
+makedepends="gtk+3-devel"
+depends="mednafen"
+short_desc="Front-end for the Mednafen emulator"
+maintainer="VladTheImpaler <vladimir-romanov-ii@yandex.com>"
+license="GPL-3.0-only"
+homepage="https://github.com/AmatCoder/mednaffe"
+distfiles="https://github.com/AmatCoder/${pkgname}/archive/${version}.tar.gz"
+checksum=ec5b3db0d83d747ce6ad0c3b0cf4483914b2308aa089ec52dd9d45d5b093acb7
+pre_configure() {
+	autoconf
```

Comment:
Why `autoconf` in `pre_configure()` ?

According to [README](https://github.com/AmatCoder/mednaffe/blob/master/README#L15) you just have to run:
````
./configure --enable-gtk3
make
make install
````